src/Pandell.PackageRunner/AssemblyChecker.cs

```diff
@@ -9,17 +8,33 @@ namespace PackageRunner
     /// </summary>
     internal class AssemblyChecker
     {
-        [DllImport("mscoree.dll", CharSet = CharSet.Unicode)]
-        private static extern bool StrongNameSignatureVerificationEx(string wszFilePath, byte fForceVerification, ref byte pfWasVerified);
+        private delegate bool VerifySignatureDelegate(string assemblyPath, bool forceCheck, out bool wasVerified);
```

Comment:
Please move the `private` members to the bottom of the class, after a [private separator](https://github.com/pandell/Pli/blob/v4.6.0.213/Source/Pandell.Common/ObjectExtensions.cs#L99-L101).